pkgs/top-level/python-packages.nix

```
@@ -1272,6 +1272,8 @@ self: super: with self; {
 
   beaker = callPackage ../development/python-modules/beaker { };
 
+  before_after = callPackage ../development/python-modules/before_after { };
```

```suggestion
  before-after = callPackage ../development/python-modules/before-after { };
```
we follow the package name normalization PEP, see python docs for details

---

Fixed